src/pages/User.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+
+export default function Profile(){
+  const location = useLocation();
+  const userList = location.state?.userList;
+
+  return(
+   <section>
+     <h1 className='text-xl mb-5'>User 정보</h1>
+      <ul>
+        {userList &&

```

Comment:
얘는 이렇게 줄일 수 있을 것 같아요!

```suggestion
        {userList?.map((user) => <li key={user.id}>{user.id} : {user.email}</li>)}
```